.jenkins/pytorch/build.sh

```diff
@@ -17,12 +17,22 @@ fi
 COMPACT_JOB_NAME="${BUILD_ENVIRONMENT}-build"
 source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
 
+if [[ "$BUILD_ENVIRONMENT" == *pytorch-linux-xenial-cuda* ]]; then
+  if which conda; then
+    conda uninstall -y cmake
+  fi
+  sudo apt-get install -y --no-install-recommends cmake=3.5\*
```

Comment:
@ezyang is this the way you'd recommend doing this versus making changes in the jobs repo?